Server/src/Server.Infrastructure/Persistence/Configurations/ReviewConfiguration.cs

```diff
@@ -0,0 +1,21 @@
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+using Server.Domain.Entities;
+
+namespace Server.Infrastructure.Persistence.Configurations;
+
+public class ReviewConfiguration
+{
+    public void Configure(EntityTypeBuilder<Review> builder)
+    {
+        builder.HasKey(x => x.Id);
+
+        builder.Property(x => x.Title)
+            .HasMaxLength(250);
+
+        builder.Property(x => x.Description)
```

Comment:
w sumie możemy to dać opcjonalne, jak się coś ludziom podoba to nie chcą poświecać czasu na pisanie o tym że coś jest git